client/js/components/day/employee.js

```diff
@@ -7,12 +9,14 @@ var Employee = React.createClass({
     // rating: rating of employee as number
     return {
       name: '',
-      rating: ''
+      rating: '',
+      gigID: Infinity,
+      employeeID: Infinity
     };
   },
 
   render: function(){
-    return (
+    return this.props.connectDragSource(
```

Comment:
Im a little surprised the connectDragSource is a method on the props of the ReactComponent. Did you have to pass that in somewhere or does the DragSource module hook that up for us?